src/native/corehost/fxr/fx_muxer.cpp

```diff
@@ -473,6 +474,17 @@ namespace
             }
         }
 
+        std::vector<std::pair<pal::string_t, pal::string_t>> additional_properties;
+        if (is_sdk_command)
+        {
+            pal::dll_t fxr;
+            pal::string_t fxr_path;
+            pal::get_loaded_library(LIBFXR_NAME, "hostfxr_main", &fxr, &fxr_path);
```

Comment:
You can use `get_own_module_path`, since this is in `hostfxr` now (should be faster too).